lib/plugins.js

```diff
@@ -3,7 +3,7 @@ import customInputRules from "./plugins/custom-input-rules";
 import customKeymap from "./plugins/custom-keymap";
 import editorGovspeakClass from "./plugins/editor-govspeak-class";
 import linkTooltip from "./plugins/link-tooltip";
-import menu from "./plugins/menu";
+import menu from "./plugins/menu/menu.js";
```

Comment:
Do you need the `.js` and in the other files?

Reply:
no i don't, my intelliJ added them by default - can remove